tests/test_core.py

```diff
@@ -6246,6 +6245,13 @@ def test_dyncall_specific(self):
       self.set_setting('EXTRA_EXPORTED_RUNTIME_METHODS', exported_runtime_methods)
       self.do_run_in_out_file_test('tests', 'core', 'dyncall_specific.c')
 
+      self.set_setting('EXTRA_EXPORTED_RUNTIME_METHODS', [])
+      self.emcc_args = emcc_args + ['-s', 'DYNCALLS=1', '--js-library', path_from_root('tests', 'core', 'test_dyncalls.js')]
+      self.do_run_in_out_file_test('tests', 'core', 'test_dyncalls.c')
+
+      self.emcc_args += ['-s', 'MINIMAL_RUNTIME=1', '-s', 'DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=["$dynCall"]']
+      self.do_run_in_out_file_test('tests', 'core', 'test_dyncalls.c')
```

Comment:
Sorry, I'm not sure sure what is going on with this test?

Why why where these lines added to this test rather than creating a new one?  This test has a loop over 3 different modes but these 2 new `do_run_in_out_file_test` don't seem to use the loop variables at all.  

Should the first part be it own test alongside `test_dyncalls_minimal_runtime` below?   The second part looks identical to `test_dyncalls_minimal_runtime`.. unless I'm missing something?

Reply:
> Why why where these lines added to this test rather than creating a new one?

@kripken asked me in https://github.com/emscripten-core/emscripten/pull/13296#pullrequestreview-576749490 to merge these two tests together. Good point about misplacing the tests inside the loop, I'll follow up with a PR to fix.